docs/containers.md

```diff
@@ -0,0 +1,98 @@
+# Containers
+
+Samples can be added to a number of different containers, which can be extended by your plugins.
```

Comment:
I'm not entirely sure if the low level implementation detail, which the `Item` is will help them to understand the usage of the system. I think this might be clearer to start with. But I'm sure we'll have to revisit this!